etc/bundler-node.conf.sample

```diff
@@ -1,4 +1,5 @@
-mnemonic = ""
+mnemonic = "test test test test test test test test test test test junk"
```

Comment:
Can we kept this empty? multiple nodes with the same mnemonic cause a bunch of weird issues that are hard to detect. With the current code, if the `mnemonic` is empty the node will generate a random one that's only kept in memory.

To avoid the risk of users sending funds and attempting to use these temporal mnemonics for "relaying" I also made it so if `num_senders > 0` and the mnemonic is random, the service should panic.

Reply:
yes, good point. updated.